src/ray/raylet/lib/java/org_ray_runtime_raylet_RayletClientImpl.cc

```diff
@@ -59,23 +59,17 @@ JNIEXPORT jlong JNICALL Java_org_ray_runtime_raylet_RayletClientImpl_nativeInit(
  * Signature: (J[BLjava/nio/ByteBuffer;II)V
  */
 JNIEXPORT void JNICALL Java_org_ray_runtime_raylet_RayletClientImpl_nativeSubmitTask(
-    JNIEnv *env, jclass, jlong client, jbyteArray cursorId, jbyteArray taskSpec) {
+    JNIEnv *env, jclass, jlong client, jbyteArray taskSpec) {
```

Comment:
It seems that you didn't change the header file. There's an instruction on how to do that in `RayletClientImpl.java`.